docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,69 @@
 [//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
 
+# Prepare your project to connect to a local deployment
```

Comment:
and get rid of the todo at the top as we don't mention mobile_launch.json anymore